Makefile

```diff
@@ -163,6 +163,11 @@ com: .gopath
 	@sh -c "PATH=$(PATHX) $(GOBUILD) -o com ./go/cmd/com"
 	$(FIXRPATH) com
 
+bpf: .gopath
+	@echo "$(GOBUILD) -o bpf ./go/cmd/bpf"
+	@sh -c "PATH=$(PATHX) $(GOBUILD) -o bpf ./go/cmd/bpf"
+	$(FIXRPATH) bpf
+
```

Comment:
We're ending up with a lot of commands... there *must* be a better way than just copy pasting this a hundred times :D (I was considering doing a busybox-style combined command for the main binary)

Reply:
I suck at makefiles, but could we do a sub on ALL_TARGETS .to make a phony target, and then

```make
%: %.bla .gopath
	@echo "$(GOBUILD) -o $@ ./go/cmd/$@"
	@sh -c "PATH=$(PATHX) $(GOBUILD) -o $@ ./go/cmd/$@"+	$(FIXRPATH) $@
```